topbeat/beat/config.go

```diff
@@ -4,9 +4,10 @@ type TopConfig struct {
 	Period *int64
 	Procs  *[]string
 	Stats  struct {
-		System     *bool
-		Proc       *bool
-		Filesystem *bool
+		System       *bool
+		Proc         *bool
+		Filesystem   *bool
+		Cpu_per_core *bool
```

Comment:
Can you use here `CpuPerCore *bool yaml:"cpu_per_core" to make sure we get the right config? We should also add it for the above 3 configs as we did in the other beats to make it clear, the values are coming from a yaml doc.

In the example above the backticks are missing, as they are also use to mark some code :-( here is an example: https://github.com/elastic/filebeat/blob/master/config/config.go#L39